src/app/utils/item.js

```diff
@@ -38,6 +38,7 @@ function LibraryItem() {
         const id = item['@id'].substring(4);
         let status = item.status && item.status[0].prefLabel ? item.status[0].prefLabel : '';
         let availability = status.replace(/\W/g, '').toLowerCase();
+        let accessMessage = item.accessMessage && item.accessMessage[0].prefLabel ? item.accessMessage[0].prefLabel.toLowerCase() : '';
```

Comment:
`item.accessMessage && item.accessMessage.length` is a better check. JS can't tell if `item.accessMessage` is an array and if it's not, then it will blow up when it tries to do `item.accesMessage[0]` on a non-array variable.